packages/ra-data-simple-prisma/src/defaultHandler.ts

```diff
@@ -62,6 +63,7 @@ export const defaultHandler = async (
     }
     case "create": {
       return await createHandler(req as CreateRequest, res, model, {
+        ...options?.create,
```

Comment:
does this work if options is undefined? 🤔

Reply:
yup